Tests/Acceptance/DoctrineAccessTokenManagerTest.php

```diff
@@ -109,45 +232,79 @@ public function testClearExpiredWithRefreshToken(): void
             timecop_return();
         }
 
+        $em->clear();
+
+        $mapFunction = function (RefreshToken $refreshToken): string {
+            return $refreshToken->getIdentifier();
+        };
+
         $this->assertSame(
-            $testData['output'],
-            $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC'])
+            array_map($mapFunction, $testData['output']),
+            array_map($mapFunction, $em->getRepository(RefreshToken::class)->findBy(['accessToken' => null], ['identifier' => 'ASC']))
         );
     }
 
-    private function buildClearExpiredTestDataWithRefreshToken(Client $client): array
+    public function testClearRevokedWithRefreshToken(): void
     {
-        $validRefreshTokens = [
-            $this->buildRefreshToken('1111', '+1 day', $client),
-            $this->buildRefreshToken('2222', '+1 hour', $client),
-            $this->buildRefreshToken('3333', '+1 second', $client),
-            $this->buildRefreshToken('4444', 'now', $client),
-        ];
+        /** @var EntityManagerInterface $em */
+        $em = $this->client->getContainer()->get('doctrine.orm.entity_manager');
+        $doctrineAccessTokenManager = new DoctrineAccessTokenManager($em);
 
-        $expiredRefreshTokens = [
-            $this->buildRefreshToken('5555', '-1 day', $client),
-            $this->buildRefreshToken('6666', '-1 hour', $client),
-            $this->buildRefreshToken('7777', '-1 second', $client),
-        ];
+        $client = new Client('client', 'secret');
+        $em->persist($client);
+        $em->flush();
 
-        return [
-            'input' => array_merge($validRefreshTokens, $expiredRefreshTokens),
-            'output' => $expiredRefreshTokens,
-        ];
+        timecop_freeze(new DateTimeImmutable());
```

Comment:
@franjo-zadelj-trikoder time freezing is not needed for revoked tokens (here and in other places too).